src/commands/Utility/language.ts

```diff
@@ -63,24 +63,22 @@ const command: Command = {
 		let language = interaction.options.getString("language", ["set", "stats"].includes(subCommand))
 
 		if (subCommand === "list") {
-			const langList: string[] = [],
-				authorLanguage = (await collection.findOne({ id: interaction.user.id }))!.lang ?? transformDiscordLocale(interaction.locale)
-			botLocales.forEach(async (element, index, array) => {
-				if (element === "empty" && !member?.roles.cache.has(ids.roles.admin)) return
-				let languageString: string
-				if (element === "empty") languageString = "Empty"
-				else languageString = getString(element)
-				langList.push(authorLanguage === element ? `**${languageString}**` : languageString)
-				if (index === array.length - 1) {
-					const embed = new EmbedBuilder({
-						color: colors.neutral,
-						author: { name: getString("moduleName") },
-						title: getString("listTitle"),
-						description: langList.join(", "),
+			const authorLanguage = (await collection.findOne({ id: interaction.user.id }))!.lang ?? transformDiscordLocale(interaction.locale),
```

Comment:
make sure to import client from index
```suggestion
			const authorLanguage = (await client.getUser(interaction.user.id)).lang ?? transformDiscordLocale(interaction.locale),
```